content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -664,31 +699,42 @@ Allocated resources:
   680m (34%)      400m (20%)    920Mi (11%)        1070Mi (13%)
 ```
 
-In the preceding output, you can see that if a Pod requests more than 1120m
-CPUs or 6.23Gi of memory, it will not fit on the node.
+In the preceding output, you can see that if a Pod requests more than 1.120 CPUs,
+or more than 6.23Gi of memory, that Pod will not fit on the node.
 
-By looking at the `Pods` section, you can see which Pods are taking up space on
+By looking at the “Pods” section, you can see which Pods are taking up space on
 the node.
 
 The amount of resources available to Pods is less than the node capacity, because
-system daemons use a portion of the available resources. The `allocatable` field
-[NodeStatus](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#nodestatus-v1-core)
-gives the amount of resources that are available to Pods. For more information, see
-[Node Allocatable Resources](https://git.k8s.io/community/contributors/design-proposals/node/node-allocatable.md).
-
-The [resource quota](/docs/concepts/policy/resource-quotas/) feature can be configured
-to limit the total amount of resources that can be consumed. If used in conjunction
-with namespaces, it can prevent one team from hogging all the resources.
-
-### My Container is terminated
-
-Your Container might get terminated because it is resource-starved. To check
-whether a Container is being killed because it is hitting a resource limit, call
+system daemons use a portion of the available resources. Within the Kubernetes API,
+each Node has a `.status.allocatable` field
+(see [NodeStatus](/docs/reference/kubernetes-api/cluster-resources/node-v1/#NodeStatus)
+for details).
+
+The `.status.allocatable` field describes the amount of resources that are available
+to Pods on that node (for example: 15 virtual CPUs and 7538 MiB of memory).
+For more information on node allocatable resources in Kubernetes, see
+[Reserve Compute Resources for System Daemons](/docs/tasks/administer-cluster/reserve-compute-resources/).
+
+You can configure [resource quotas](/docs/concepts/policy/resource-quotas/)
+to limit the total amount of resources that a namespace can consume.
+Kubernetes enforces quotas for objects in particular namespace when there is a
+ResourceQuota in that namespace.
+For example, if you assign specific namespaces to different teams, you
+can set resource quotas on those namespaces to prevent one team from
+hogging all the resources in your cluster.
```

Comment:
```suggestion
using all the resources in your cluster.
```

Reply:
Here hogging and using are not synonyms. You can “hog all the resources” whilst leaving too little for other teams; “using all the resources” implies that nobody else got any at all.

Maybe there's another phrasing that works better?

Reply:
I think the word "all" also adds to that potential confusion. 

Hmm. Something like "avoid situations where one team uses too many resources in the cluster, leaving other teams with too little"? Could use some more massaging :P